react-native/providers/AuthProvider.js

```diff
@@ -105,7 +105,7 @@ export const AuthProvider = props => {
     const fetchCommunities = async () => {
         await AsyncStorage.getItem('communities').then(communityData => {
             console.log('Fetching communties from AsyncStorage: ');
-            if (communityData) {
+            if (false) {
```

Comment:
This won't execute. Is this intentional for now?

Reply:
Oh yeah. Good catch. Was just trying always fetching communities. Not meant to be a permanent change.